src/styles/components/_commander-option.scss

```diff
@@ -1,12 +1,24 @@
 .c-commander-option {
     border: 0;
     padding: 0;
-    background-color: $color-black;
-    color: $color-white;
+    background-color: rgb(51 38 1 / 17%);
+    color: $color-black;
+    width: 250px;
```

Comment:
I've been using `rem` instead of `px` at a scale of 10px to 1 rem, so that it works correctly when zooming in and out.  Would you be able to make this change?